### PR TITLE
Gulag mining scanners can not disarm gibtonite

### PR DESCRIPTION
**Pull request by silicons**
I believe the argument was it was too slow otherwise, but I don't think prisoners should have easy access to explosives either.

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
i was about to complain that this would stop freegolems, but you did it right for once.

**Comment by Buggy123** [comment]
:-1:  Emergent gameplay: if they can get bombs, prisoners might actually do something other than escape/die trying to escape/suicide immediately.

**Comment by Repukan** [comment]
There's nothing emergent about prisoners being able to get free explosives that can potentially gib.

The only emergent thing is officers and wardens shoving buckshot up your ass so they don't have to deal with bombed shuttles.

**Comment by ShiggyDiggyDo** [comment]
Giving prisoners mining scanners was a mistake

**Comment by Iamgoofball** [comment]
lmao just make it disarm and dissolve gibtonite

**Comment by Hatfish** [comment]
remind me why it was imperative that a punishment role needed a QoL change again?

**Comment by Arzion5** [comment]
Here is why you are wrong forever : 

- If prisoners acquire explosives, they still can't do shit, if they break into the cockpit of the prisoner shuttle and use the explosive then they'll get spaced and die quickly once they board the station
- They literally can't do shit with gibtonite in any other way, thanks to barbaric mongolian ching chong japanese hindu tribal TG laws, if the prisoner gets perma'd but manages to kill whoever imprisoned them, then the prisoner gets punished. If the prisoner was unjustly permabrigged, they should have ahelped. Thanks TG server rules, you are truly an inspiration for us all! 😄

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -223,6 +223,11 @@
 		return
 	if(primed)
 		if(istype(I, /obj/item/device/mining_scanner) || istype(I, /obj/item/device/t_scanner/adv_mining_scanner) || istype(I, /obj/item/device/multitool))
+			if(istype(I, /obj/item/device/mining_scanner))
```
Comment (Cyberboss) [inline code/modules/mining/ores_coins.dm]: don't do the istype twice

**Review by Cyberboss — DISMISSED** [review]

**Comment by 81Denton** [comment]
@Arzion5 If they can't do shit with it, why do you want gulagites to have free gibtonite then 🤔

**Comment by Arzion5** [comment]
If they can't do shit, why do you want to nerf mining scanners?

**Comment by kevinz000** [comment]
because you don't even play tg more than a little every few months and frankly i don't think you should be complaining if you don't even play.
🤡 
other than that because prisoners shouldn't get free bombs, deal with it.

**Comment by Arzion5** [comment]
"If I place an emote after insulting the opposition before I use an argument, I will sound smart and hilarious!"
Relax, it's christmas.
It will be a stretch to call the PR grudgecoding after a prisoner managed to escape, blow up something or someone with gibtonite, or both, but there is simply no way that the gulag mining scanner would be nerfed as a precaution, it is too unnoticeable to be a pre-cautionary change. It leads to the conclusion that this PR was made in the wake of a prisoner using gibtonite. Besides, I have already presented the already-existing flaws and dangers when prisoners use gibtonite, so the nerf is just salt on a wound.

**Comment by JJRcop** [comment]
Maybe this scanner shouldn't even detect gibtonite

**Comment by Bawhoppen** [comment]
Merge this posthaste. Don't give more hugbox stuff to prisoners.

**Comment by DoctoreMad** [comment]
@Bawhoppen Are you projecting your need for hugbox onto the prisoners or something?

**Comment by Buggy123** [comment]
Shit like this is why I always help free gulag prisoners. Always. Even shuttle them back to the station and give them a spare mining jumpsuit.

**Comment by optimumtact** [comment]
you know what, no

**Comment by Bawhoppen** [comment]
@DoctoreMad It is a matter of perspective which side is more hugbox

**Pull request closed without merging**